package.json

```diff
@@ -31,33 +31,36 @@
     "@amsterdam/design-system-react": "^0.13.1",
     "@amsterdam/design-system-react-icons": "^0.1.13",
     "@amsterdam/design-system-tokens": "^0.13.0",
-    "next": "^15.0.2",
+    "@eslint/eslintrc": "^3.1.0",
+    "@eslint/js": "^9.14.0",
+    "next": "^15.0.3",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-hook-form": "^7.53.2",
     "sharp": "^0.33.5"
   },
   "devDependencies": {
-    "@next/eslint-plugin-next": "^14.2.17",
+    "@next/eslint-plugin-next": "^15.0.3",
+    "@types/eslint__js": "^8.42.3",
     "@types/react": "^18.3.12",
     "@types/react-dom": "^18.3.1",
-    "@typescript-eslint/eslint-plugin": "^7.18.0",
-    "@typescript-eslint/parser": "^7.18.0",
-    "eslint": "^8.57.0",
+    "eslint": "^9.14.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^18.0.0",
-    "eslint-config-next": "^13.5.7",
+    "eslint-config-next": "^15.0.3",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-prettier": "^5.2.1",
     "eslint-plugin-react": "^7.37.2",
     "eslint-plugin-react-hooks": "^4.6.2",
-    "npm-check-updates": "17.1.10",
+    "npm-check-updates": "17.1.11",
     "plop": "^4.0.1",
     "prettier": "^3.3.3",
-    "typescript": "^5.6.3"
+    "typescript": "^5.6.3",
+    "typescript-eslint": "^8.14.0"
   },
   "overrides": {
+    "eslint": "$eslint",
```

Comment:
This line prevents me from installing the packages.

> Error: Unable to resolve reference $eslint

I’m on node 20.18.0, have removed node_modules and ran npm install.

Installing does succeed after changing it to `"eslint": "^9.14.0",` as per [this comment](https://github.com/vitejs/vite/issues/17832#issuecomment-2273849150).

Reply:
Running a dev build then presents an error in next.js:

> Hydration failed because the server rendered HTML didn't match the client. As a result this tree will be regenerated on the client. This can happen if a SSR-ed Client Component used

But that could be one of my browser extensions: it doesn’t occur in a private window.